src/portal_visualization/builders/anndata_builders.py

```diff
@@ -50,7 +62,7 @@ def get_conf_cells(self, marker=None):
         if f'{zarr_path}/.zgroup' not in file_paths_found:
             message = f'RNA-seq assay with uuid {self._uuid} has no .zarr store at {zarr_path}'
             raise FileNotFoundError(message)
-        vc = VitessceConfig(name=self._uuid)
+        vc = VitessceConfig(name=self._uuid, schema_version='1.0.15')
```

Comment:
Can we define `schema_version` somewhere for reuse?

Reply:
Great call! I've added this as a private field in the `BaseBuilder` which allows for overrides to be done (if ever needed) by passing `schema_version` to the args of any builder.